codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/SyncClientClass.java

```diff
@@ -138,14 +139,26 @@ private MethodSpec constructorWithAdvancedConfiguration() {
         List<MethodSpec> methods = new ArrayList<>();
         ClassName returnType = poetExtensions.getModelClass(opModel.getReturnType().getReturnType());
 
-        methods.add(SyncClientInterface.operationMethodSignature(model, opModel)
+        methods.add(SyncClientInterface.operationMethodSignature(model, opModel, opModel.getMethodName())
                                   .addAnnotation(Override.class)
                                   .addCode(getCustomResponseHandler(opModel, returnType)
                                                .orElseGet(() -> protocolSpec.responseHandler(opModel)))
                                   .addCode(protocolSpec.errorResponseHandler(opModel))
                                   .addCode(protocolSpec.executionHandler(opModel))
                                   .build());
 
+        // Add extra methods for paginated operations
+        if (opModel.isPaginated()) {
+            methods.add(SyncClientInterface.operationMethodSignature(model, opModel,
```

Comment:
If new lining, we typically new line all arguments

Reply:
Done